src/main/kotlin/org/kamiblue/client/util/NBSReader.kt

```diff
@@ -0,0 +1,84 @@
+package org.kamiblue.client.util
+
+import org.kamiblue.client.module.modules.misc.NoteBot
+import java.io.DataInputStream
+import java.io.File
+import java.io.FileInputStream
+import java.util.*
+import kotlin.collections.ArrayList
+
+internal object NBSReader {
+
+    fun readNbs(fileName: String): TreeMap<Long, ArrayList<NoteBot.Note>> {
```

Comment:
Maybe keep this in NoteBot instead of making a one-time used util

Reply:
Ok.